.goreleaser.yml

```diff
@@ -32,7 +32,7 @@ changelog:
     - '^test:'
 brews:
   - name: trellis-cli
-    github:
+    tap:
```

Comment:
Do we need to update goreleaser? Or, is it broken for a long time?

Reply:
They often update/deprecate config options so I'm guessing they changed this after the last release we did. As far as I know every release has worked fine (but won't from now on until this fix).